metadata.json

```diff
@@ -17,6 +17,7 @@
   "dependencies": [
     {"name": "puppetlabs/stdlib", "version_requirement": ">= 4.2.0 < 5.0.0"},
     {"name": "puppetlabs/apt", "version_requirement": ">= 2.1.1 < 3.0.0"},
+    {"name": "puppet/archive", "version_requirement": ">= 1.1.2 < 3.0.0"},
```

Comment:
Are we using archive?

Reply:
I can merge if you remove this.

Reply:
Now we have a few new errors from existing code. This would go away anyway if you merge #15, I would guess.